cmd/statusd/utils.go

```diff
@@ -1488,3 +1492,25 @@ func testValidateNodeConfig(t *testing.T, config string, fn func(common.APIDetai
 
 	fn(resp)
 }
+
+func testProfiling(t *testing.T) bool {
+	if err := common.StartCPUProfile(""); err != nil {
+		t.Error("Cannot start profiling:", err)
+		return false
+	}
+
+	address, _, _, err := statusAPI.CreateAccount(TestConfig.Account1.Password)
+	if err != nil {
+		t.Errorf("could not create account: %v", err)
+		return false
+	}
+
+	t.Log("Account created: ", address)
+
+	if err := common.StopCPUProfile(); err != nil {
+		t.Error("Cannot stop profiling:", err)
+		return false
+	}
+
```

Comment:
What about assertions? I'd like to at least check that profiling files are presented on the filesystem and have non-zero size.